experimental/ner_spancat_compare/.gitignore

```diff
@@ -0,0 +1,3 @@
+assets
+corpus
+training
```

Comment:
Should we add `metrics` here? Or did you want to include the metrics from an example run in the project for some reason?

Reply:
Ah yeah I added metrics to gitignore now. I added it just as a form of habit! I want to "save" the results for myself